.github/workflows/heroku-deploy.yml

```diff
@@ -0,0 +1,43 @@
+# Your workflow name.
+name: Deploy to heroku.
+
+# Run workflow on every push to master branch.
+on:
+  push:
+    branches: [main]
+
+# Your workflows jobs.
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      # Check-out the repository.
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Build, Push and Release a Docker container to Heroku.
+        uses: gonuit/heroku-docker-deploy@v1.3.3 # GitHub action name
+        with:
+          # Variables for your Heroku app.
+
+          # The email address associated with your Heroku account.
+          email: ${{ secrets.HEROKU_EMAIL }}
+
+          # Heroku API key associated with provided user's email.
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+
+          # Name of the heroku application to which the build is to be sent.
+          heroku_app_name: ${{ secrets.HEROKU_APP_NAME }}
+
+          # Dockerfile directory.
+          dockerfile_directory: ./
+
+          # Dockerfile name.
+          dockerfile_name: Dockerfile
+
+          # Additional options of docker build command.
+          docker_options: "--no-cache"
+
```

Comment:
We also have a docker-compose file that executes the web server, how do we integrate it with heroku?

Reply:
I read on their docs, they state the definition of a Procfile, do we need to add it here on the repository?

Reply:
The Procfile is used to specify process types, we won't have to create this file since we are using the `web` process and the action used in this YAML file has this property as default.

Regarding the docker-compose, Heroku currently offers no support to docker-compose deploys, their offer to replace this is the usage of [heroku.yml](https://devcenter.heroku.com/articles/build-docker-images-heroku-yml) files. However, I think that deploying the Docker image will satisfy our needs.

Reply:
What about this?

<img width="791" alt="image" src="https://user-images.githubusercontent.com/61894952/191747781-5d3aa28a-5729-45d0-a4ca-55f519a43c64.png">

Reply:
Since we are using a Docker image the Github Action will start the Docker container using the Dockerfile, so if we want to change to command to start the app the change will be made in the Dockerfile.